ui/app/components/ui/mascot/mascot.stories.js

```diff
@@ -0,0 +1,98 @@
+import EventEmitter from 'events'
+import React, { useState } from 'react'
+import Button from '../button'
+import ButtonGroup from '../button-group'
+import Mascot from './mascot.component'
+
+const animationEventEmitter = new EventEmitter()
+
+const containerStyle = {
+  height: '600px',
+  width: '357px',
+  border: '1px solid black',
+  display: 'flex',
+  flexFlow: 'column',
+  justifyContent: 'center',
+  alignItems: 'center',
+}
+
+const buttonStyle = {
+  marginTop: '16px',
+}
+
+export default {
+  title: 'Mascot',
+}
+
+export function Approve () {
```

Comment:
Was this name ("Approve") intentional? I don't get it :thinking:  Here's another suggested name:

```suggestion
export function demo () {
```